common/src/main/java/org/apache/comet/parquet/NativeBatchReader.java

```diff
@@ -236,7 +236,7 @@ public NativeBatchReader(AbstractColumnReader[] columnReaders) {
   public void init() throws Throwable {
 
     conf.set("spark.sql.parquet.binaryAsString", "false");
-    conf.set("spark.sql.parquet.int96AsTimestamp", "false");
+    conf.set("spark.sql.parquet.int96AsTimestamp", "true");
```

Comment:
I still have questions about these configs. In the unikely event that a user explicity sets `spark.sql.parquet.int96AsTimestamp=false`, shouldn't we respect that?

Reply:
https://github.com/apache/datafusion-comet/issues/1816

Reply:
Rethinking this along with investigating the ci failures